app/src/components/UI/Button.css

```diff
@@ -0,0 +1,20 @@
+.btn{
+  width: 176px;
+  height: 47px;
+  margin: auto 3rem ;
+  color: aliceblue;
+  background: #58864C;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
things like box shadows can be an exception to that pixel vs rem rule though, @Ebrahim-Merchant can confirm.